### PR TITLE
csv sql example

### PR DESCRIPTION
**Pull request by gthrone25**
New page and tweaking an includes to point at table manage now that it's live (and to start moving away from ingest table). We'll figure out logical buckets/folders for this in the near future but getting content out there for users.

### CODE REVIEW

**Comment by gthrone25** [comment]
@ljsinclair Let me know if you're good with this. It will act as the structure I intend for many more SQL3 tutorials :)

File: sql-preview/_posts/2022-08-25-sql-bulk-insert-csv.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: How do I Bulk Insert A CSV File?
+---
+
+## Before you begin
+{% include /sql-preview/before_ingest.md %}
+
+## Step 0: learn about the data
+
+1. Learn a little bit about the [age dataset](https://www.kaggle.com/datasets/imoore/age-dataset?resource=download)
```
Comment (ljsinclair) [inline sql-preview/_posts/2022-08-25-sql-bulk-insert-csv.md:10]: First thing, there is never a step 0. Step 0 is "before you begin" :)
https://molecula.atlassian.net/wiki/spaces/DOCS/pages/1010925640/How+to+write+better+documentation#Rule-0%3A-There-is-no-0-(zero)

File: sql-preview/_posts/2022-08-25-sql-bulk-insert-csv.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: How do I Bulk Insert A CSV File?
+---
+
```
Comment (ljsinclair) [inline sql-preview/_posts/2022-08-25-sql-bulk-insert-csv.md:4]: I'm missing context for this content. Does the user do this for *every* CSV file?

Comment (gthrone25) [reply sql-preview/_posts/2022-08-25-sql-bulk-insert-csv.md:4]: Updated. Maybe we need a standard title for something like this? LMK

**Comment by gthrone25** [comment]
@ljsinclair pushing some changes up. LMK if it looks good to you!

**Review by ljsinclair — APPROVED** [review]
all good.

**Pull request merged**